CONTRIBUTING.md

```diff
@@ -78,6 +87,8 @@
     * If an identifier contains a verb or an adjective in addition to a noun, the schema [verb|adjective]\[noun] is
       preferred, e.g., use `left_input` rather than ~~`input_left`~~ and `set_left_input()` rather than
       ~~`set_input_left()`~~.
+    * Unused variables: instead of leaving out the parameter name, comment the parameter name out (e.g.,
+      `void function(const size_t /*unused*/) { ... }`).
```

Comment:
The current example might suggest using `/*unused*/` in general. However, based on the discussions in #2441, I think we prefer a descriptive identifier if possible.

```suggestion
      `void function(const size_t /*value*/) { ... }`).
```